Mathlib/Algebra/Group/Hom/Instances.lean

```diff
@@ -68,7 +68,9 @@ instance MonoidHom.commGroup {M G} [MulOneClass M] [CommGroup G] : CommGroup (M
       simp,
     zpow_succ' := fun n f => by
       ext x
-      simp [zpow_coe_nat, pow_succ],
+      -- If we combine these two `simp`s we reveal a bug: "unexpected bound variable #0"
```

Comment:
Is there an issue for this? Also, the first simp is now non-terminal.

Reply:
https://github.com/leanprover-community/mathlib4/issues/11357

Reply:
I've also put the link in the comments.